client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1169,6 +1176,7 @@
   "messages.confirm-changing-from-new": "Note: changing from 'New' will remove any lines with zero quantity.",
   "messages.confirm-delete-assets_one": "This will permanently remove this asset.",
   "messages.confirm-delete-assets_other": "This will permanently remove the {{count}} selected assets.",
+  "messages.confirm-delete-bundled-item": "This will permanently delete this bundled item",
```

Comment:
This kind of implies that it will delete the item rather than "unbundling" it. Perhaps "This will permanently remove this item from the bundle", or "This will un-bundle this item" ?

Reply:
[1282793](https://github.com/msupply-foundation/open-msupply/pull/5324/commits/128279354807e9f9e7b6d3039d68463d8ecad2aa)